packages/loader/container-loader/src/deltaManager.ts

```diff
@@ -1334,7 +1334,20 @@ export class DeltaManager
     /**
      * Retrieves the missing deltas between the given sequence numbers
      */
-    private fetchMissingDeltas(telemetryEventSuffix: string, lastKnowOp: number, to?: number) {
+     private fetchMissingDeltas(telemetryEventSuffix: string, lastKnowOp: number, to?: number) {
+         // eslint-disable-next-line @typescript-eslint/no-floating-promises
+         this.fetchMissingDeltasCore(telemetryEventSuffix, false /* cacheOnly */, lastKnowOp, to);
```

Comment:
we should avoid floating promises, at least add a .catch that logs an error, otherwise partners catch these errors in their global unhandled rejection handlers

Reply:
it would also be nice if the method name expressed that this kicked off a background action, like fire or start fetchMissingDeltas

---
In reply to: [613427107](https://github.com/microsoft/FluidFramework/pull/5773#discussion_r613427107) [](ancestors = 613427107)

Reply:
It really does not throw as fetchMissingDeltasCore catches all exceptions and reports it. I changed it to include more code in try/catch. I do not like adding dead code for the sake of compiler happiness.
As for name, I can't come up with short name.. It has been named this way for years, I actually like current one, but if you have good short suggestions - let's change it in separate PR.